Mjml.Net/MjmlRenderContext.cs

```diff
@@ -62,19 +55,25 @@ public ValidationErrors Validate()
             return validator?.Complete() ?? new ValidationErrors();
         }
 
-        public void ReadFragment(TextReader mjml)
+        public void ReadFragment(string mjml)
         {
-            using (var xml = XmlReader.Create(mjml, ReaderSettings))
+            if (mjmlOptions.Lax)
             {
-                ReadXml(xml, currentComponent);
+                mjml = XmlFixer.FixXML(mjml, mjmlOptions);
```

Comment:
Minor: We could probably just call the method `XmlFixer.Fix`?

Reply:
Can change that.

Reply:
Done